### ISSUE TITLE
Update Information to Explain a bit more about limitations of Outbound Spam Policies

### ISSUE THREAD

**Issue by kbrosseau** [MicrosoftDocs/microsoft-365-docs#11791]
I have people who want to have it documented more clearly that when you have multiple Outbound AntiSpam Policies with the same scoped users/groups, that only the highest priority takes effect. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c3c27660-0f0b-3002-7183-854e4fb8716b
* Version Independent ID: 3f436862-e6b3-2a9d-70b5-f7eee02eec2e
* Content: [Configure outbound spam policies - Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/outbound-spam-policies-configure?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/outbound-spam-policies-configure.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/outbound-spam-policies-configure.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@kbrosseau, I did some updates to that article earlier this wee. The [priority section](https://learn.microsoft.com/microsoft-365/security/office-365-security/outbound-spam-policies-configure#use-the-microsoft-365-defender-portal-to-set-the-priority-of-custom-outbound-spam-policies) section now contains:

> Outbound spam protection stops for a sender after the first policy is applied. For more information, see [Order and precedence of email protection](how-policies-and-protections-are-combined.md).

That 'Order and precedence' article has existed for several years.

Is there anything else I should do to amplify?

**Comment by kbrosseau** [comment]
> @kbrosseau, I did some updates to that article earlier this wee. The [priority section](https://learn.microsoft.com/microsoft-365/security/office-365-security/outbound-spam-policies-configure#use-the-microsoft-365-defender-portal-to-set-the-priority-of-custom-outbound-spam-policies) section now contains:
> 
> > Outbound spam protection stops for a sender after the first policy is applied. For more information, see [Order and precedence of email protection](how-policies-and-protections-are-combined.md).
> 
> That 'Order and precedence' article has existed for several years.
> 
> Is there anything else I should do to amplify?

I would mention that only one AntiSpam policy will take effect for that group/users, and it's the highest priority. Maybe reword it to specify that you can't have multiple policies on the same scoped set of users/groups that monitor the same thing. I hope what I'm saying is making sense.

**Comment by chrisda** [comment]
@kbrosseau, I updated the statement. Does this help?

> Outbound spam protection stops for a sender after the first policy is applied (the highest priority policy for that sender). For more information, see [Order and precedence of email protection](how-policies-and-protections-are-combined.md).

**Comment by kbrosseau** [comment]
> @kbrosseau, I updated the statement. Does this help?
> 
> > Outbound spam protection stops for a sender after the first policy is applied (the highest priority policy for that sender). For more information, see [Order and precedence of email protection](how-policies-and-protections-are-combined.md).

I like that addition of the highest priority policy for that sender. I think that's good at the moment! I appreciate it!
